src/turbostat.c

```diff
@@ -1566,6 +1578,8 @@ turbostat_config(const char *key, const char *value)
 	} else if (strcasecmp("PackageThermalManagement", key) == 0) {
 		config_ptm = IS_TRUE(value);
 		apply_config_ptm = 1;
+	} else if (strcasecmp("LogicalCoreNames", key) == 0) {
+		config_lcn = 1;
```

Comment:
This does not behave in the way it is documented: setting `LogicalCoreNames false` will also *enable* the feature, because the argument to the option is not considered. Please use `config_lcn = IS_TRUE(value)` as above.